app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the home controller here.
```

Comment:
Do not commit empty files pls, they will be here forever. Also, we prefer to not use comments in code.